ironfish/src/webApi.ts

```diff
@@ -100,6 +108,13 @@ export class WebApi {
     return response.data.data
   }
 
+  async getUser(id: number): Promise<ApiUser | null> {
+    return await axios
+      .get<ApiUser>(`${this.host}/users/${id}`, this.options())
+      .then((r) => r.data)
+      .catch(() => null)
```

Comment:
Are there error cases which necessitate this line?

Reply:
If the user is not found you get a 404 which produces an exception